app/views/canvas/full_zoom/_my_module.html.erb

```diff
@@ -60,14 +60,12 @@
           </a>
         </li>
       <% end %>
-      <% if can_view_module_activities(my_module) %>
+      <% if can_read_experiment?(my_module.experiment) %>
```

Comment:
I don't know if will be better to have a permission for modules eg can_read_module? instead of this what do you mean?

Reply:
We split the permissions into three (actually, four with organization level) main levels: organization, team, project, experiment. So module falls under experiment.

Reply:
:+1: